src/cpp/point_cloud.cpp

```diff
@@ -11,6 +12,8 @@
 
 #include "Eigen/Dense"
 
+#define MAX(x, y) ((x) > (y) ? (x) : (y))
```

Comment:
Very minor request: could we get rid of this and use `std::max()` instead?

I realize that it is probably harmless here inside a .cpp file, but I still like to avoid this coding pattern because it can cause all kinds of trouble if it ends up in a header.

Reply:
Sure, I can use `std::max` instead. I will change the pull request later today.

I agree with this and even had a version with `std::max` but finally though that within a cpp file it's fine. But you are probably right that these patterns tend to spread :)